### PR TITLE
feat(Redis): Add redis connection.

### PR DESCRIPTION
**Pull request by yremmet**
- Adds dependecy.
- Sets redis entry to true when creating a new binding.
- Deletes redis entry when deleting an existing binding.

### CODE REVIEW

**Comment by yremmet** [comment]
@rschollmeyer Please review the reds implementation of @mberger2015

File: osb-service/src/main/java/de/evoila/cf/broker/custom/autoscaler/AutoscalerBindingService.java [hunk 1]
```diff
@@ -70,13 +103,25 @@ protected ServiceInstanceBinding bindService(String bindingId, ServiceInstanceBi
...
+            if (jedis.get(binding.getAppGuid()) != null) {
+            	// A running nozzle will automatically create a new unregistered entry in Redis for this app the next time a metric or a log is received
+                jedis.del(binding.getAppGuid());
+
+                log.info("Successfully updated the subscription status for app = " + serviceInstance.getId()
+        		+ ". Application is now not registered.");
+            } else {
+                log.error("Error updating the subscription status for app = " + binding.getAppGuid()
+                        + ". Application might still be registered.");
+            }
```
Comment (rschollmeyer) [inline osb-service/src/main/java/de/evoila/cf/broker/custom/autoscaler/AutoscalerBindingService.java:124]: jedis.close() is missing

**Comment by rschollmeyer** [comment]
Looks good to me

File: osb-service/src/main/java/de/evoila/cf/broker/custom/autoscaler/AutoscalerBindingService.java (hunk 1 above)
Comment (marberger) [reply osb-service/src/main/java/de/evoila/cf/broker/custom/autoscaler/AutoscalerBindingService.java:124]: jedis.close(); added

**Pull request merged**